### PR TITLE
call escrow.Balance when daemon starts

### PR DESCRIPTION
**Pull request by laipogo**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior?** (You can also refer to a JIRA ticket here)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **What dependencies / modules need to be updated as pre-requisites?** (Include relevant PR links here)

* **Description of changes**

---

* **Please check if the PR fulfills these requirements**
- [ ] The subject of this PR contains a JIRA ticket BTFS-xxxx (N/A for community PRs)
- [ ] PR description and commit messages are [good and meaningful](https://chris.beams.io/posts/git-commit/)
- [ ] Unit tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **Please make sure the following procedures have been applied before requesting reviewers**
- [ ] Code changes closely follow [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
- [ ] Code has been *rebased* against latest master (`git merge` not recommended, unless you know what you are doing)
- [ ] Code changes have run through `go fmt`
- [ ] Code changes have run through `go mod tidy`
- [ ] All unit tests passed locally (`make test_go_test`)

### CODE REVIEW

File: cmd/btfs/daemon.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"errors"
 	_ "expvar"
 	"fmt"
+	"github.com/TRON-US/go-btfs/core/escrow"
```
Comment (jin-tron) [inline cmd/btfs/daemon.go]: please update the import order

File: cmd/btfs/daemon.go [hunk 2]
```diff
@@ -311,6 +313,21 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 		return err
 	}
 
+	// init ledger account and get balance
+	go func() {
+		bo := backoff.WithMaxRetries(backoff.NewExponentialBackOff(), 3)
+		err := backoff.Retry(func() error {
+			balance, err := escrow.Balance(req.Context, cfg)
+			if err == nil {
+				fmt.Println("banlance:", balance)
```
Comment (jin-tron) [inline cmd/btfs/daemon.go]: maybe can change to "ledger balance"

File: cmd/btfs/daemon.go [hunk 3]
```diff
@@ -311,6 +313,21 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
...
+			}
+			return err
+		}, bo)
+		if err != nil {
+			log.Error("Get balance failed.")
```
Comment (taiyangc) [inline cmd/btfs/daemon.go]: Change to "Unable to get Ledger balance. Please check network settings and restart daemon."

Comment (taiyangc) [reply cmd/btfs/daemon.go]: Log the error as well.

File: cmd/btfs/daemon.go [hunk 4]
```diff
@@ -4,6 +4,8 @@ import (
 	"errors"
 	_ "expvar"
 	"fmt"
+	"github.com/TRON-US/go-btfs/core/escrow"
+	"github.com/cenkalti/backoff"
```
Comment (taiyangc) [inline cmd/btfs/daemon.go]: use backoff/v3

**Pull request closed without merging**